mitiq/calibration/calibrator.py

```diff
@@ -140,23 +142,25 @@ def log_results_flat(self) -> None:
         return print(tabulate(table, headers, tablefmt="simple_grid"))
 
     def log_results_cartesian(self) -> None:
-        """Prints calibration results in the following form
-        ┌──────────────────────────────┬────────────────────────────┬────────────────────────────┐
-        │ strategy\benchmark           │ Type: rb                   │ Type: ghz                  │
-        │                              │ Num qubits: 2              │ Num qubits: 2              │
-        │                              │ Circuit depth: 337         │ Circuit depth: 2           │
-        │                              │ Two qubit gate count: 80   │ Two qubit gate count: 1    │
-        ├──────────────────────────────┼────────────────────────────┼────────────────────────────┤
-        │ Technique: ZNE               │ ✔                          │ ✘                          │
-        │ Factory: Richardson          │ Noisy error: 0.1128        │ Noisy error: 0.0117        │
-        │ Scale factors: 1.0, 2.0, 3.0 │ Mitigated error: 0.0501    │ Mitigated error: 0.0439    │
-        │ Scale method: fold_global    │ Improvement factor: 2.2515 │ Improvement factor: 0.2665 │
-        ├──────────────────────────────┼────────────────────────────┼────────────────────────────┤
-        │ Technique: ZNE               │ ✔                          │ ✘                          │
-        │ Factory: Richardson          │ Noisy error: 0.1128        │ Noisy error: 0.0117        │
-        │ Scale factors: 1.0, 3.0, 5.0 │ Mitigated error: 0.0408    │ Mitigated error: 0.0171    │
-        │ Scale method: fold_global    │ Improvement factor: 2.7672 │ Improvement factor: 0.6852 │
-        └──────────────────────────────┴────────────────────────────┴────────────────────────────┘
+        """
+        Prints the results of the calibrator in cartesian form::
+
+            ┌──────────────────────────────┬────────────────────────────┬────────────────────────────┐
+            │ strategy\\benchmark           │ Type: rb                   │ Type: ghz                  │
+            │                              │ Num qubits: 2              │ Num qubits: 2              │
```

Comment:
Exactly! The extra slash is also the reason for the `|` to be over one space in the `calibrator.py` file. However, when printing the docsstring in the API docs, it all lines up correctly since only one `\` is printed.